packages/@okta/vuepress-site/docs/reference/api/threat-insight/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: ThreatInsight
+category: management
+---
+
+# ThreatInsight Configuration API
+
+[Okta ThreatInsight](https://help.okta.com/en/prod/Content/Topics/Security/security-threat-insight.htm) maintains a constantly evolving list of IPs that exhibit suspicious behaviors suggestive of malicious activity. Authentication requests associated with an IP in this list can be logged in [System Log](https://help.okta.com/en/prod/Content/Topics/Reports/Reports_SysLog.htm?cshid=ext_Reports_SysLog) and blocked. The Okta ThreatInsight Configuration API provides operations to manage your ThreatInsight configuration.
```

Comment:
Here is the alias for the help.okta.com link. Better to use this than just the URL for future changes:

https://help.okta.com/en/prod/okta_help_CSH.htm#ext_threatinsight 

Also:

Second sentence:

can be -> are

Authentication requests associated with an IP in this list **are** logged in.........

Reply:
Some context: ThreatInsight can operate in three modes: disabled, log only, and log and block. Thus the term `can be` since in some modes requests are not blocked.